test/k8sT/Policies.go

```diff
@@ -1661,19 +1677,35 @@ var _ = Describe("K8sPolicyTest", func() {
 				})
 
 				It("Validates fromEntities remote-node policy", func() {
-					By("Installing default-deny ingress policy")
-					importPolicy(kubectl, testNamespace, cnpDenyIngress, "default-deny-ingress")
-
-					By("Checking that remote-node is disallowed by default")
-					validateNodeConnectivity(HostConnectivityAllow, RemoteNodeConnectivityDeny)
+					installDefaultDenyIngressPolicy()
 
 					By("Installing fromEntities remote-node policy")
 					importPolicy(kubectl, testNamespace, cnpFromEntitiesRemoteNode, "from-entities-remote-node")
 
 					By("Checking policy correctness")
-					validateNodeConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow)
+					validateConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow, PodConnectivityDeny)
 				})
 			})
+
+			It("Validates fromEntities cluster policy", func() {
+				installDefaultDenyIngressPolicy()
+
+				By("Installing fromEntities cluster policy")
+				importPolicy(kubectl, testNamespace, cnpFromEntitiesCluster, "from-entities-cluster")
+
+				By("Checking policy correctness")
+				validateConnectivity(HostConnectivityAllow, RemoteNodeConnectivityAllow, PodConnectivityAllow)
+			})
+
+			It("Validates fromEntities all policy", func() {
+				installDefaultDenyIngressPolicy()
+
+				By("Installing fromEntities cluster policy")
+				importPolicy(kubectl, testNamespace, cnpFromEntitiesAll, "from-entities-all")
```

Comment:
By text should reflect that the policy is `fromEntities` `all` vs. `cluster`, right now the text is the same for both cases.

Reply:
Oh... 🤦‍♂️
Thanks! I'll fix it.